ssl/statem/extensions_clnt.c

```diff
@@ -524,12 +528,57 @@ int tls_construct_ctos_psk_kex_modes(SSL *s, WPACKET *pkt, X509 *x,
     return 1;
 }
 
-int tls_construct_ctos_key_share(SSL *s, WPACKET *pkt, X509 *x, size_t chainidx,
-                                 int *al)
+#ifndef OPENSSL_NO_TLS1_3
+static int add_key_share(SSL *s, WPACKET *pkt, unsigned int curve_id)
+{
+    unsigned char *encodedPoint = NULL;
+    EVP_PKEY *key_share_key = NULL;
```

Comment:
We should figure out style on initializing.  But since ```key_share_key``` is set in the very first statement of the function, and then encodedPoint, I would remove these.  Not going to argue about it strongly, however.  But, I do think encodedPoint is better written as ```encoded_point``` since we don't do camelCase.